tags/environmentalsustainability.md

```diff
@@ -0,0 +1,54 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we will first identify how to reduce the environmental footprints of increasingly large data centers and then help the community take action to mitigate these footprints.
+
+References:
+
+* https://www.iea.org/reports/net-zero-by-2050
+* https://www.whitehouse.gov/wp-content/uploads/2021/10/US-Long-Term-Strategy.pdf
+* https://www.worldbank.org/en/news/feature/2022/05/23/what-you-need-to-know-about-net-zero
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.  This TAG will identify values and possible incentives for service providers to reduce their consumption and carbon footprint through cloud native tooling.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- the trade off between reducing resource consumption and higher performance is hard to balance
+- there is a lack of well supported and documented approaches for evaluating cloud native projects in their default or optimized configurations
```

Comment:
```suggestion
- there is a lack of well supported and documented approaches for evaluating the efficiency of cloud native projects in their default or optimized configurations
```

... just because we're not talking, say, fitness for purpose here.

Reply:
@jberkus  this changes my interpretation of the sentence.  For instance, we may not care about the efficiency in some cases, but may care more about workload positioning, in a cross-cloud instance (transfer between sunny regions with solar power around the world, for instance).  Are you okay with leaving the wording as is, with that consideration?  Or do you have another suggestion?

Reply:
@jberkus Alternative wording proposal that I think may address the intent behind proposing to add "efficiency" but without being overly limiting as per @catblade's point. 
```suggestion
- there is a lack of well supported and documented approaches for evaluating the environmental sustainability of cloud native projects in their default or optimized configurations

```